utils.js

```diff
@@ -8,6 +8,20 @@ export function getClientIdFromOptions(options, servicesConfig) {
   return servicesConfig.clientId;
 }
 
+export function getAppIdFromOptions(options) {
+  const { appId } = options;
+  return appId || null;
+}
+
+export function getServiceConfiguration(appId = null) {
+  const service = ServiceConfiguration.configurations.findOne({
+    service: "apple",
+  });
+  const serviceConfiguration = service[appId] || service;
```

Comment:
it's not better to throw an error if appId is provided but the service is not configured for this appId?